modal-go/proto/modal_proto/api.pb.go

```diff
@@ -6042,6 +6042,106 @@ func (b0 AutoscalerSettings_builder) Build() *AutoscalerSettings {
 	return m0
 }
 
+// Used for flash autoscaling
```

Comment:
Are these changes intentional?

Reply:
Yes, because of how go packaging works [we check in the proto files for Go](https://github.com/modal-labs/libmodal/blob/main/DEVELOPING.md#modal-go-development)! And we needed to bump the protos here for the new flag. But it makes for horrible diffs...